shared/src/main/scala/squants/package.scala

```diff
@@ -75,6 +75,33 @@ package object squants {
   type Price[A <: Quantity[A]] = squants.market.Price[A]
 
   /**
+    * Helper function to achieve uniform Double formatting over JVM and JS platforms.
+    * Simple Double.toString will format 1.0 as "1.0" on JVM and as "1" on JS
+    * @param d Double number to be formatted
+    * @return
+    */
+  private[squants] def crossFormat(d: Double): String = {
```

Comment:
Scala Native now supports formatting, so that can now be shared across platforms, but with the toString implementation on Double right now Scala Native includes trailing zeroes, which would break a lot of unit tests.